packages/shell-api/src/replica-set.spec.ts

```diff
@@ -13,6 +13,7 @@ import {
   ALL_TOPOLOGIES,
   asShellResult
 } from './enums';
+import { CliServiceProvider } from '../../service-provider-server';
```

Comment:
Is there a reason not to use 

```suggestion
import { CliServiceProvider } from '@monogsh/service-provider-server';
```

like we usually do? If it’s about the fact that it’s not in the `package.json` here, we could list it under `devDependencies`, right?

Reply:
We had issues with having circular dependency problems (even with devDependency) since the `service-provider-server package` requires the `shell-api` package. We do the same thing in `integration.spec.ts`, since the  only place we use `CliServiceProvider` in this package is in tests.